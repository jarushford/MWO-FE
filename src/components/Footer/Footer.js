import React, { Component } from 'react'
import '../../main.scss'

export default class Footer extends Component {
  constructor() {
    super()
    this.state = {
      email: '',
      error: false,
      success: false
    }
  }

  updateEmail = e => {
    this.setState({ email: e.target.value })
  }

  handleSubmit = async () => {
    const url = 'https://mwo-be.herokuapp.com/api/v1/mailing'
    const response = await fetch(url, {
      method: 'POST',
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify({ email: this.state.email })
    })

    if (!response.ok) {
      this.setState({ error: true })
    } else {
      this.setState({
        error: false,
        success: true,
        email: ''
      })
      setTimeout(() => {
        this.setState({ success: false })
      }, 4000)
    }
  }

  render() {
    return (
      <footer>
        <section className="subscribe-section">
          <h3>Subscribe for tour updates and more!</h3>
          <input
            type="text"
            placeholder="mad@wallacemail.com"
            value={this.state.email} 
            onChange={this.updateEmail}
          />
          <p className={`error-msg ${this.state.error && 'errored'}`}>
            Sorry, something went wrong! Feel free to try again later.
          </p>
          <p className={`success-msg ${this.state.success && 'success'}`}>
            Added Successfully, yay!
          </p>
          <button className="btn rounded" onClick={this.handleSubmit}>
            <span className="text-green">Subscribe</span>
          </button>
          <div className="social-container">
            <a href="https://www.facebook.com/madwallaceband/" target="_blank" rel="noopener noreferrer">
              <i className="fab fa-facebook-square" />
            </a>
            <a href="https://www.instagram.com/mad.wallace/" target="_blank" rel="noopener noreferrer">
              <i className="fab fa-instagram" />
            </a>
            <a href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7" target="_blank" rel="noopener noreferrer">
              <i className="fab fa-spotify" />
            </a>
            <a href="mailto:madwallaceband@gmail.com">
              <i className="fas fa-envelope" />
            </a>
            <a href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA" target="_blank" rel="noopener noreferrer">
              <i className="fab fa-youtube" />
            </a>
            <a href="https://www.youtube.com/watch?v=_lvB2tHSEG4" target="_blank" rel="noopener noreferrer">
              <i className="fas fa-bath" />
            </a>
          </div>
        </section>
        <section className="sub-footer">
          <img src="./assets/blue-logo.png" alt="blue mad wallace logo" />
          <p>copyright Â© 2019</p>
        </section>
      </footer>
    )
  }
}