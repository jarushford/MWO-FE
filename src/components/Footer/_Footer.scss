.subscribe-section {
  background: url('./assets/jake-battle.jpg');
  height: 290px;
  background-size: 1050px;
  background-position: center;
  background-color: rgba(0,0,0,0.35);
  background-blend-mode: darken;
  margin: auto;
  padding: 0 20px;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  align-items: center;
  border-top: 3px solid #b999c8;
  border-bottom: 3px solid #b999c8;
  margin-top: 75px;
  position: relative;
  text-align: center;

  h3 {
    color: white;
    letter-spacing: 1.5px;
    font-size: 1.4rem;
    margin: 0;
    text-shadow: 0px 3px 4px rgba(0,0,0,0.4),
                 0px 8px 18px rgba(0,0,0,0.1),
                 0px 18px 23px rgba(0,0,0,0.1);
  }

  input {
    width: 320px;
    height: 45px;
    border-radius: 25px;
    padding-left: 15px;
    font-size: 1.1rem;
    font-family: 'Open Sans', sans-serif;
    border: none;
    outline: none;
    box-sizing: border-box;
    transition: 2s ease;

    &:focus {
      border: 2px solid $purple;
      box-shadow: 0 0  8px 0px $purple;
    }

    &::placeholder {
      opacity: .5;
    }
  }
}

.btn{
	margin: 20px auto;
	border: none;
	padding: 10px 34px;
	font-size: 1.2rem;
  position: relative;
  background: black;
  outline: none;
	&::before{
		transition: all 0.85s cubic-bezier(0.68, -0.55, 0.265, 1.55);
		content: '';
		width: 50%;
		height: 100%;
		position: absolute;
		top: 0;
		left: 0;
	}

	& .btn-text{
		mix-blend-mode: overlay;
	}
	&:hover{
		&::before{
			width: 100%;
		}
	}

	&.rounded{
    border-radius:50px;
		& .text-green{
			color: blue;
			mix-blend-mode: overlay;
		}
		
		&::before{
			border-radius: 50px;
			width: 25%;
      background: $purple;
      box-shadow: 0 0  8px 0px $purple;
      animation: move 8s ease infinite;
		}
		&:hover{
			&::before{
			background: $purple;
      width: 100%;
      -webkit-animation: 0;
      animation: 0;
			}
		}	
	}
}

@keyframes move {
  0%, 100% {
    margin-left: 0
  }
  50% {
    margin-left: 138px
  }
}

.sub-footer {
  margin: 20px auto;
  width: 100%;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-around;

  p {
    color: white;
    font-family: 'Open Sans', sans-serif;
  }

  img {
    width: 90px;
  }
}

.error-msg,
.success-msg {
  color: tomato;
  text-shadow: 0px 3px 4px rgba(0,0,0,0.4),
                 0px 8px 18px rgba(0,0,0,0.1),
                 0px 18px 23px rgba(0,0,0,0.1);
  position: absolute;
  top: 120px;
  opacity: 0;
}

.success-msg {
  color: rgb(137, 221, 80)
}

.errored,
.success {
  opacity: 1;
}

.social-container {
  width: 300px;
  display: flex;
  justify-content: space-between;

  i {
    font-size: 2rem;
    color: #f5cc8f;
    transition: .3s ease;

   &:hover {
      transform: translateY(-2px);
    }
  }
}

@media screen and (min-width: 1050px) {
  .subscribe-section {
    background-size: 100%;
  }
}