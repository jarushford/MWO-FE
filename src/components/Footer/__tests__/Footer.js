import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../Footer'

jest.useFakeTimers()

describe('Footer', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Footer />)
  })

  it('should match the default snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})