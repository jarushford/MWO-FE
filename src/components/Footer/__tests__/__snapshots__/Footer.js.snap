// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the default snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section
          className="subscribe-section"
        >
          <h3>
            Subscribe for tour updates and more!
          </h3>
          <MailchimpSubscribe
            render={[Function]}
            url="https://madwallace.us17.list-manage.com/subscribe/post?u=f8fb81c993f8ecc4aaa9dde81&amp;id=2824a93352"
          />
          <div
            className="social-container"
          >
            <a
              href="https://www.facebook.com/madwallaceband/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-facebook-square"
              />
            </a>
            <a
              href="https://www.instagram.com/mad.wallace/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-instagram"
              />
            </a>
            <a
              href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-spotify"
              />
            </a>
            <a
              href="mailto:madwallaceband@gmail.com"
            >
              <i
                className="fas fa-envelope"
              />
            </a>
            <a
              href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-youtube"
              />
            </a>
            <a
              href="https://www.youtube.com/watch?v=_lvB2tHSEG4"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fas fa-bath"
              />
            </a>
          </div>
        </section>,
        <section
          className="sub-footer"
        >
          <img
            alt="blue mad wallace logo"
            src="./assets/blue-logo.png"
          />
          <p>
            Â© Mad Wallace 2020
          </p>
        </section>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Subscribe for tour updates and more!
            </h3>,
            <MailchimpSubscribe
              render={[Function]}
              url="https://madwallace.us17.list-manage.com/subscribe/post?u=f8fb81c993f8ecc4aaa9dde81&amp;id=2824a93352"
            />,
            <div
              className="social-container"
            >
              <a
                href="https://www.facebook.com/madwallaceband/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-facebook-square"
                />
              </a>
              <a
                href="https://www.instagram.com/mad.wallace/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-instagram"
                />
              </a>
              <a
                href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-spotify"
                />
              </a>
              <a
                href="mailto:madwallaceband@gmail.com"
              >
                <i
                  className="fas fa-envelope"
                />
              </a>
              <a
                href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-youtube"
                />
              </a>
              <a
                href="https://www.youtube.com/watch?v=_lvB2tHSEG4"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fas fa-bath"
                />
              </a>
            </div>,
          ],
          "className": "subscribe-section",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Subscribe for tour updates and more!",
            },
            "ref": null,
            "rendered": "Subscribe for tour updates and more!",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "render": [Function],
              "url": "https://madwallace.us17.list-manage.com/subscribe/post?u=f8fb81c993f8ecc4aaa9dde81&amp;id=2824a93352",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <a
                  href="https://www.facebook.com/madwallaceband/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-facebook-square"
                  />
                </a>,
                <a
                  href="https://www.instagram.com/mad.wallace/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-instagram"
                  />
                </a>,
                <a
                  href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-spotify"
                  />
                </a>,
                <a
                  href="mailto:madwallaceband@gmail.com"
                >
                  <i
                    className="fas fa-envelope"
                  />
                </a>,
                <a
                  href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-youtube"
                  />
                </a>,
                <a
                  href="https://www.youtube.com/watch?v=_lvB2tHSEG4"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fas fa-bath"
                  />
                </a>,
              ],
              "className": "social-container",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fab fa-facebook-square"
                  />,
                  "href": "https://www.facebook.com/madwallaceband/",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fab fa-facebook-square",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fab fa-instagram"
                  />,
                  "href": "https://www.instagram.com/mad.wallace/",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fab fa-instagram",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fab fa-spotify"
                  />,
                  "href": "https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fab fa-spotify",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fas fa-envelope"
                  />,
                  "href": "mailto:madwallaceband@gmail.com",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fas fa-envelope",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fab fa-youtube"
                  />,
                  "href": "https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fab fa-youtube",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fas fa-bath"
                  />,
                  "href": "https://www.youtube.com/watch?v=_lvB2tHSEG4",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fas fa-bath",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
            ],
            "type": "div",
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="blue mad wallace logo"
              src="./assets/blue-logo.png"
            />,
            <p>
              Â© Mad Wallace 2020
            </p>,
          ],
          "className": "sub-footer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "blue mad wallace logo",
              "src": "./assets/blue-logo.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Â© Mad Wallace 2020",
            },
            "ref": null,
            "rendered": "Â© Mad Wallace 2020",
            "type": "p",
          },
        ],
        "type": "section",
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="subscribe-section"
          >
            <h3>
              Subscribe for tour updates and more!
            </h3>
            <MailchimpSubscribe
              render={[Function]}
              url="https://madwallace.us17.list-manage.com/subscribe/post?u=f8fb81c993f8ecc4aaa9dde81&amp;id=2824a93352"
            />
            <div
              className="social-container"
            >
              <a
                href="https://www.facebook.com/madwallaceband/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-facebook-square"
                />
              </a>
              <a
                href="https://www.instagram.com/mad.wallace/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-instagram"
                />
              </a>
              <a
                href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-spotify"
                />
              </a>
              <a
                href="mailto:madwallaceband@gmail.com"
              >
                <i
                  className="fas fa-envelope"
                />
              </a>
              <a
                href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-youtube"
                />
              </a>
              <a
                href="https://www.youtube.com/watch?v=_lvB2tHSEG4"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fas fa-bath"
                />
              </a>
            </div>
          </section>,
          <section
            className="sub-footer"
          >
            <img
              alt="blue mad wallace logo"
              src="./assets/blue-logo.png"
            />
            <p>
              Â© Mad Wallace 2020
            </p>
          </section>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Subscribe for tour updates and more!
              </h3>,
              <MailchimpSubscribe
                render={[Function]}
                url="https://madwallace.us17.list-manage.com/subscribe/post?u=f8fb81c993f8ecc4aaa9dde81&amp;id=2824a93352"
              />,
              <div
                className="social-container"
              >
                <a
                  href="https://www.facebook.com/madwallaceband/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-facebook-square"
                  />
                </a>
                <a
                  href="https://www.instagram.com/mad.wallace/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-instagram"
                  />
                </a>
                <a
                  href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-spotify"
                  />
                </a>
                <a
                  href="mailto:madwallaceband@gmail.com"
                >
                  <i
                    className="fas fa-envelope"
                  />
                </a>
                <a
                  href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-youtube"
                  />
                </a>
                <a
                  href="https://www.youtube.com/watch?v=_lvB2tHSEG4"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fas fa-bath"
                  />
                </a>
              </div>,
            ],
            "className": "subscribe-section",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Subscribe for tour updates and more!",
              },
              "ref": null,
              "rendered": "Subscribe for tour updates and more!",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "render": [Function],
                "url": "https://madwallace.us17.list-manage.com/subscribe/post?u=f8fb81c993f8ecc4aaa9dde81&amp;id=2824a93352",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <a
                    href="https://www.facebook.com/madwallaceband/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <i
                      className="fab fa-facebook-square"
                    />
                  </a>,
                  <a
                    href="https://www.instagram.com/mad.wallace/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <i
                      className="fab fa-instagram"
                    />
                  </a>,
                  <a
                    href="https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <i
                      className="fab fa-spotify"
                    />
                  </a>,
                  <a
                    href="mailto:madwallaceband@gmail.com"
                  >
                    <i
                      className="fas fa-envelope"
                    />
                  </a>,
                  <a
                    href="https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <i
                      className="fab fa-youtube"
                    />
                  </a>,
                  <a
                    href="https://www.youtube.com/watch?v=_lvB2tHSEG4"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <i
                      className="fas fa-bath"
                    />
                  </a>,
                ],
                "className": "social-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fab fa-facebook-square"
                    />,
                    "href": "https://www.facebook.com/madwallaceband/",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fab fa-facebook-square",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fab fa-instagram"
                    />,
                    "href": "https://www.instagram.com/mad.wallace/",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fab fa-instagram",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fab fa-spotify"
                    />,
                    "href": "https://open.spotify.com/artist/6H2CmXWubR7YVd6n1hlXq7",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fab fa-spotify",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fas fa-envelope"
                    />,
                    "href": "mailto:madwallaceband@gmail.com",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fas fa-envelope",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fab fa-youtube"
                    />,
                    "href": "https://www.youtube.com/channel/UC6N29y6h0Fqh7VmEObN6LHA",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fab fa-youtube",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fas fa-bath"
                    />,
                    "href": "https://www.youtube.com/watch?v=_lvB2tHSEG4",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fas fa-bath",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
              ],
              "type": "div",
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="blue mad wallace logo"
                src="./assets/blue-logo.png"
              />,
              <p>
                Â© Mad Wallace 2020
              </p>,
            ],
            "className": "sub-footer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "blue mad wallace logo",
                "src": "./assets/blue-logo.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Â© Mad Wallace 2020",
              },
              "ref": null,
              "rendered": "Â© Mad Wallace 2020",
              "type": "p",
            },
          ],
          "type": "section",
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
